pkg/client/client_test.go

```diff
@@ -1719,8 +1723,10 @@ func (s *ClientTestSuiteFM) TestIsFeatureEnabledWithDecisionError() {
 		ConfigManager:   s.mockConfigManager,
 		DecisionService: s.mockDecisionService,
 	}
+
+	// should still return the decision because the error is non-fatal
 	result, err := client.IsFeatureEnabled(testFeature.Key, testUserContext)
-	s.False(result)
+	s.True(result)
 	s.NotNil(err)
```

Comment:
We're returning an `err` _and_ a decision? I thought we wanted to swallow these errors if they're not fatal. Was this this missed or intentionally kept?

Also we should change to either `s.Error(err)` or s`.NoError(err)` for consistency :)

Reply:
I missed this. We shouldn't return the error. Thanks for catching!

Reply:
the only true error returned is from GetProjectConfig() method;  getFeatureDecision() has that call (indirectly). if we have an error from GetProjectConfig() then I still think we need to return it - that's a legit error.